examples/tutorials/analysis-3d/energy_dependence_estimation.py

```diff
@@ -0,0 +1,266 @@
+
+Setup
+-----
+
+"""
+
+from astropy import units as u
+from astropy.coordinates import SkyCoord
+from astropy.table import Table
+from regions import CircleSkyRegion
+from gammapy.data import DataStore
+from gammapy.datasets import MapDataset
+from gammapy.estimators.energydependence import (
+    EnergyDependenceEstimator,
+    weighted_chi2_parameter,
+)
+from gammapy.makers import (
+    DatasetsMaker,
+    FoVBackgroundMaker,
+    MapDatasetMaker,
+    SafeMaskMaker,
+)
+from gammapy.maps import MapAxis, WcsGeom
+from gammapy.modeling.models import (
+    GaussianSpatialModel,
+    PowerLawSpectralModel,
+    SkyModel,
+)
+from gammapy.stats.utils import ts_to_sigma
+
+######################################################################
+# Check setup
+# -----------
+
+# from gammapy.utils.check import check_tutorials_setup
+# check_tutorials_setup()
+
+######################################################################
+# Obtain the data to use
+# ----------------------
+#
+# Create the data store and obtain the observations from the HESS DL3 DR1 for MSH 1552.
+#
+# P.S.: do not forget to setup your environment variable $GAMMAPY_DATA to your local directory
+
+data_store = DataStore.from_dir("$GAMMAPY_DATA/hess-dl3-dr1")
+obs_id = data_store.obs_table["OBS_ID"][data_store.obs_table["OBJECT"] == "MSH 15-5-02"]
+observations = data_store.get_observations(obs_id)
+
+######################################################################
+# Setting the exclusion mask
+# --------------------------
+#
+# First, we define the energy range to obtain the dataset within. The geometry is also
+# defined, based on the position of MSH 1552 (the source of interest here).
+
+energy_axis = MapAxis.from_energy_bounds(0.2, 100, nbin=15, unit="TeV")
+energy_axis_true = MapAxis.from_energy_bounds(
+    0.05, 110, nbin=30, unit="TeV", name="energy_true"
+)
+
+source_pos = SkyCoord(320.33, -1.19, unit="deg", frame="galactic")
+geom = WcsGeom.create(
+    skydir=(source_pos.galactic.l.deg, source_pos.galactic.b.deg),
+    frame="galactic",
+    axes=[energy_axis],
+    width=5,
+    binsz=0.02,
+)
+regions = CircleSkyRegion(center=source_pos, radius=0.7 * u.deg)
+exclusion_mask = geom.region_mask(regions, inside=False)
+exclusion_mask.sum_over_axes().plot()
+
+######################################################################
+# Data reduction loop
+# -------------------
+#
+# For details on how the data reduction is performed see the
+# :doc:`/tutorials/analysis-3d/analysis_3d` tutorial.
+#
+# The data reduction steps can be combined using the DatasetsMaker class
+# which takes as an input the list of makers.
+# Here we stack the dataset in this step.
+#
+
+safe_mask_maker = SafeMaskMaker(
+    methods=["aeff-default", "offset-max"], offset_max=2.5 * u.deg
+)
+
+dataset_maker = MapDatasetMaker()
+
+fov_bkg_maker = FoVBackgroundMaker(method="fit", exclusion_mask=exclusion_mask)
+
+global_dataset = MapDataset.create(geom)
```

Comment:
Don't you need to put the energy_axis_true here ?

```suggestion
global_dataset = MapDataset.create(geom, energy_axis_true=energy_axis_true)
```